addons_engine/addons-manager/tests/addons_service_test.py

```diff
@@ -0,0 +1,57 @@
+import sys
+from pathlib import Path
+
+import docker
+import pytest
+import tests.container_utils as container_utils
+
+current_dir = Path(__file__).parent
+parent_dir = current_dir.parent
+sys.path.append(str(parent_dir))
+
+from services.addons_service import (  # noqa: E402
```

Comment:
If I am not mistaken:
```suggestion
from services.addons_runner import (  # noqa: E402
```

Reply:
Oh yes! why does vscode not realize the refactor change 😠 Pycharm usually does.

Reply:
Yeah, sometimes its a bit flaky 😅